ManatiCR/FDI
# FDI-33 Reportes creation

[FDI-33](https://manati.atlassian.net/browse/FDI-33)
-
**Steps to test**
- Run `ahoy drush cim -y && ahoy drush cr`
- The categories and subcategories were not defined so create some, and remember the id of each taxonomy term, you would need it when running the graphql query
- Go to http://fdi.docker/es/graphql/explorer and add the following query to create a reporte node:

Query: 
```
mutation ($input: ReporteInput!) {
  createReporte(input: $input) {
    entity {
      entityId
      fieldCodigoDeSeguimient
    }
    errors
    violations {
      message
    }
  }
}
```
Query variable:
Replace the numbers in `field_categoria_reporte` and `field_subcategoria_reporte` with the id's of the taxonomoy terms you just created
```
{
  "input": {
    "title": "Reporte ejemplo 1",
    "field_correo": "luke@gmail.com",
    "body": "Mensaje del reporte",
    "field_subcategoria_reporte": [19,20],
    "field_categoria_reporte": [22, 23, 24, 21],
    "field_latitud": 44234234234423423,
    "field_longitud": 34234234234423423,
    "field_solicita_asesoria_o_apoyo": "si",
    "field_lugar": "Lugar"
  }
}
```
- After running this query a the field código de seguimiento should be prompt as a response
- Go to http://fdi.docker/es/admin/fdi/reportes and check the page looks alright
